package.json

```diff
@@ -105,7 +105,8 @@
   },
   "resolutions": {
     "braces": "^2.3.1",
-    "hosted-git-info": ">=3.0.8"
+    "hosted-git-info": ">=3.0.8",
+    "trim": "^0.0.3"
```

Comment:
Do we need resolution for this? Can you try removing this from resolution? I think it should work as `yarn.lock` is changed.

Reply:
If I remove resolution and do `yarn install`, it goes back to 0.0.1 because there is no `^`

Reply:
Yeah, why libs do this...